apps/src/templates/instructions/TopInstructions.jsx

```diff
@@ -700,9 +700,7 @@ class TopInstructions extends Component {
         />
         <div style={[isCollapsed && isCSDorCSP && commonStyles.hidden]}>
           <div style={instructionsContainerStyle} id="scroll-container">
-            <div ref={ref => this.setInstructionsRef(ref)}>
```

Comment:
Setting the reference here seemed problematic because it competed with the desired one [here](https://github.com/code-dot-org/code-dot-org/pull/46325/files#diff-51c7e711d775bd3a59807c2b790c8880a9f3345a136b86409301b220e262c3a3L510).  In some debugging, I observed the reference being repeatedly set between the two locations.